Patches/Star Wars - Factions/Patch_Apparel.xml

```diff
@@ -51,7 +51,7 @@
 				defName="PJ_ImpUniformBlack"
```

Comment:
This uniform is a soft officer's uniform, not really armored for combat. Per the star wars wiki, they're made of "gaberwool," which doesn't seem to have an special protective properties as far as I can tell.